build-ci/new_package.rst

```diff
@@ -21,8 +21,42 @@ then needs to be added to the :file:`ups/*.table` file (and possibly the
 :file:`ups/*.cfg` file) of one or more other packages in the stack where it is
 used.
 
+.. _lfs-repos:
+
+Handling ``git lfs`` backed repos
+=================================
+
+New `git lfs` (see :doc:`/tools/git_lfs`) backed repos (or existing repos
+being converted to `lfs`) require additional configuration.
+
+- The `repos.yaml`_ entry must declare that the repository is LFS backed.
+
+.. code-block:: yaml
+
+    afwdata:
+      url: https://github.com/lsst/afwdata.git
+      lfs: true
+
+  See the comment block at the top of `repos.yaml`_ for additional details.
+
```

Comment:
You might want to indent the `code-block` directive by two spaces so that it's aligned with the list item. E.g.

```
- The `repos.yaml`_ entry must declare that the repository is LFS backed.

  .. code-block:: yaml
  
     afwdata:
       url: https://github.com/lsst/afwdata.git
       lfs: true

  See the comment block at the top of `repos.yam`_ for additional details.
```